native/cocos/renderer/gfx-gles3/GLES3GPUContext.cpp

```diff
@@ -109,8 +109,11 @@ bool GLES3GPUContext::initialize(GLES3GPUStateCache *stateCache, GLES3GPUConstan
     }
 
     EGL_CHECK(eglBindAPI(EGL_OPENGL_ES_API));
-
+#if CC_EDITOR
```

Comment:
any other choice? CC_EDITOR in engine make editor and engine glue with each other

Reply:
editor's code is already glue with engine，not just here,maybe later we can find a better way to do this.